services-custom/dynamodb-enhanced/src/main/java/software/amazon/awssdk/enhanced/dynamodb/model/QueryEnhancedRequest.java

```diff
@@ -117,10 +120,10 @@ public Expression filterExpression() {
     }
 
     /**
-     * Returns the projection expression set on this request object, or null if it doesn't exist.
+     * Returns the list of projected properties on this request object, or an empty list if no projection is specified.
```

Comment:
Can we reword this to say 'projected attributes' instead of 'projected properties' which I think will be more consistent with how we refer to them everywhere else.